DataProcessing/datax-keyvault/pom.xml

```diff
@@ -67,7 +67,7 @@ SOFTWARE
     <dependency>
       <groupId>org.json4s</groupId>
       <artifactId>json4s-jackson_2.11</artifactId>
-      <version>4.0.4</version>
+      <version>3.5.3</version>
```

Comment:
What will this do to nova prod?

Reply:
Suman will help me test in nova prod

Reply:
I was able to reproduce the error in dddatax backup cluster using the latest bits of master branch. After downgrading json4s the error is gone and the job is not crashing anymore. I suspect some recent change in version in another artifact re-introduced this error after 4.0.4 upgrade. We are completing the testing and i am approving this PR after that